setup.py

```diff
@@ -18,11 +18,12 @@
         'unification',
     ],
     extras_require={
-        'test': ['flake8', 'pytest>=4.1', 'torchvision==0.2.1'],
+        'test': ['flake8', 'pytest>=4.1', 'pytest-xdist==1.27.0', 'torchvision==0.2.1'],
         'dev': [
             'flake8',
             'isort',
             'pytest>=4.1',
+            'pytest-xdist==1.27.0',
```

Comment:
why not `>=`?

Reply:
Travis was complaining with `pytest>=4.1` and `pytest-xdist>=1.27.0`, so I had to either use `pytest-xdist==1.27.0` or `pytest>=4.4`.